gravitee-apim-console-webui/src/management/api/endpoints-v4/endpoint/api-endpoint.component.spec.ts

```diff
@@ -30,8 +30,34 @@ import { ApiEndpointModule } from './api-endpoint.module';
 import { ApiEndpointHarness } from './api-endpoint.harness';
 
 import { CONSTANTS_TESTING, GioTestingModule } from '../../../../shared/testing';
-import { ApiV4, fakeApiV4, fakeConnectorPlugin } from '../../../../entities/management-api-v2';
-import { fakeEndpointGroupV4 } from '../../../../entities/management-api-v2/api/v4/endpointGroupV4.fixture';
+import { ApiV4, fakeApiV4, fakeConnectorPlugin, fakeProxyApiV4 } from '../../../../entities/management-api-v2';
+import { fakeEndpointGroupV4, fakeHTTPProxyEndpointGroupV4 } from '../../../../entities/management-api-v2/api/v4/endpointGroupV4.fixture';
+
+const healthCheckSchema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  properties: {
+    dummy: {
+      title: 'dummy',
+      type: 'string',
+      description: 'A dummy string',
+      readOnly: true,
+    },
+  },
+  required: ['dummy'],
+};
+/**
+ * Expect that a single PUT request has been made which matches the specified URL
```

Comment:
```suggestion
 * Expect that a single GET request has been made which matches the specified URL
```
But maybe we can remove the comment as it does not bring much value

Reply:
agree it does not bring much value, it's removed now